man/subscription-manager.8

```diff
@@ -271,7 +277,7 @@ Shortcut for "release --set=VERSION"
 .SS UNREGISTER OPTIONS
 The
 .B unregister
-command removes a system's subscriptions and removes it from the subscription management service.
+command does two important things. Firstly, it will implicitly remove all of the currently attached subscriptions thereby returning the consumed quantity of entitlements back to their subscription pools making them available for other consumers. Secondly, it will remove the system's consumer identity thereby removing it's contact with the currently configured subscription management service.
```

Comment:
typo "it's contact" -> "its contact"

Reply:
My bad!  I cut and pasted it assuming the grammar was correct!  Fixing